doc/manual/utilities.rst

```diff
@@ -3,25 +3,43 @@ Utilities
 
 .. Sort these tool by some subjective combination of their
    typical sequence and expected frequency of use.
+
+ARTIQ Firmware Service (AFWS) client
+------------------------------------
+
+This tool serves as a client for building tailored firmware and gateware from M-Lab's servers and downloading the binaries in ready-to-flash format. It is necessary to have a valid subscription to AFWS. Subscription also includes general helpdesk support and can be purchased or extended by contacting ``sales@``. One year of support is included with most hardware purchased from M-Labs. 
```

Comment:
To use it, it is necessary to have a valid subscription to the service.

"Most hardware" means carriers, Kaslis. Most purchases are for complete crates that include them yes, but purchasing other cards does not include support beyond initial help with setup.